src/main/java/com/tobmistaketracker/detector/VerzikP3MistakeDetector.java

```diff
@@ -102,6 +104,8 @@ public void afterDetect() {
         activeWebTiles.removeAll(webTilesToRemove);
         webTilesToRemove.clear();
         playerNamesPurpled.clear();
+        verzikP3NPC = null;
```

Comment:
I'm surprised this is working for you, because it seems like we're incorrectly clearing out the NPC here, which means you never end up calling `setVerzikAttackInfo` onGameTick. Perhaps that isn't even needed then? I think the NPC reference already updates its location, so we can remove onGameTick altogether probably. However, we shouldn't clear this out, in case we ever want it for something else.

Reply:
If you want, you can set it to null on `shutdown` instead.

Also, `shutdown` should probably call emptyTickData()